.github/workflows/chart-version-bump.yml

```diff
@@ -0,0 +1,50 @@
+## Used on Renovate PRs to bump the chart version and add a changelog entry
+## Reference: https://github.com/stefanzweifel/git-auto-commit-action
+## Reference: https://github.com/marketplace/actions/changed-files
+name: 'Chart Version Bump and Changelog'
+
+on:
+  pull_request:
+    types:
+      - labeled
+
```

Comment:
We need to set the permissions of the new workflow. Otherwise we get problems with the security score.

Do you know what permissions are needed @tico24? Example permissions:
- https://github.com/argoproj/argo-helm/blob/2c046f9a9a647ff1b53a81c0defcc311af95974d/.github/workflows/lint-and-test.yml#L5-L6
- https://github.com/argoproj/argo-helm/blob/2c046f9a9a647ff1b53a81c0defcc311af95974d/.github/workflows/pr-sizing.yml#L11-L13
- all other workflow files living in `/.github/..`

Reply:
Oh yes. Probably. 

I normally self host renovate and this isn't an option for self hosters.

Reply:
Found it:

> ```yaml
>     permissions:
>       # Give the default GITHUB_TOKEN write permission to commit and push the changed files back to the repository.
>       contents: write
> ```
> Ref: https://github.com/stefanzweifel/git-auto-commit-action

Reply:
those permissions work for me, so I have added them here.